reconcile/utils/ocm_base_client.py

```diff
@@ -89,23 +89,17 @@ def get_paginated(
             params_copy = params.copy()
         params_copy["size"] = max_page_size
 
-        # fetch pages
-        records_seen = 0
         while True:
```

Comment:
max pages could also have a default of, say 1000 and the condition could be added to the loop? :thinking: